.github/workflows/nightly.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       run_unit_tests: true
       run_integration_tests: true
       upload_clio_server: true
-      disable_cache: true
+      disable_cache: ${{ github.event_name != 'pull_request' }}
```

Comment:
It is theoretically possible that because of this condition workflow will pass CI in PR but fail when running without cache